pkg/loop/internal/pb/ccipocr3/models.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ccipocr3;ccipocr3pb";
+
+package loop.internal.pb.ccipocr3;
+
+// CCIPMsg is a gRPC adapter to [github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3.CCIPMsg]
+message CCIPMsg {
+  uint64 sequence_number = 1;
+  BigInt chain_fee_limit = 2;
+  uint64 nonce = 3;
+  uint64 chain_fee_price = 4;
+  bytes message_id = 5; // Hash [32]byte
+  uint64 source_chain_selector = 6;
+  string sender = 7; // Address
+  string receiver = 8; // Address
+  bool strict = 9;
+  string fee_token = 10; // Address
+  BigInt fee_token_amount = 11;
+  bytes data = 12;
+  repeated TokenAmount token_amounts = 13;
+  repeated bytes source_token_data = 14;
+}
+
+// BigInt represents a [big.Int].
+message BigInt {
+  bool negative = 1;
+  bytes value = 2;
```

Comment:
Is this `bool` needed? Doesn't `BigInt`'s `Bytes()` method use 2's complement to encode negative/non-negative? I think just `bytes` is enough?

Reply:
agreed